components/delete_post_modal.jsx

```diff
@@ -69,6 +69,7 @@ export default class DeletePostModal extends React.Component {
 
     handleHide() {
         this.setState({show: false});
+        document.getElementById('post_textbox').focus();
```

Comment:
This shouldn't be done if `UserAgent.isMobile()` is true (where UserAgent is from `import * as UserAgent from 'utils/user_agent';`)